lib/osf-components/addon/components/cedar-metadata-editor/component.ts

```diff
@@ -38,5 +42,8 @@ export default class CedarMetadataEditor extends Component<Args> {
         // @ts-ignore
         record.metadata = cee.currentMetadata;
         await record.save();
+        // eslint-disable-next-line no-debugger
+        // debugger;
+        this.router.transitionTo('guid-node.metadata.detail', record.id);
```

Comment:
Does this mean that this will transition to the detail route even if the save fails? Do we want to add any error handling here?